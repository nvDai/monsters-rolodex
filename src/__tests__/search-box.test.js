import React from 'react'

import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import SearchBox from '../components/search-box/search-box.component'

describe('SearchBox', () => {
  it('should render without crashing', () => {
    const mockProps = {
      placeholder: 'Type here', 
      handleChangeInput: jest.fn(),
    }
    const wrapper = shallow(<SearchBox {...mockProps} />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})