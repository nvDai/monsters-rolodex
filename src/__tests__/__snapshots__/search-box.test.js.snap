// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox should render without crashing 1`] = `
<input
  onChange={[MockFunction]}
  placeholder="Type here"
  type="search"
/>
`;
